source/frontend/src/hooks/layer-api/useLayerQuery.tsx

```diff
@@ -100,7 +100,7 @@ export const useLayerQuery = (
   withCredentials?: boolean,
 ): IUserLayerQuery => {
   const baseAllUrl = `${url}&srsName=EPSG:4326`;
-  const baseUrl = `${url}&srsName=EPSG:4326&count=1`;
+  const baseUrl = `${url}&srsName=EPSG:4326`;
```

Comment:
this seems to be affecting other methods that are only designed to fetch one result - ie. findOneWhereContainsWrapped

Reply:
Thats correct. The findoneWherecontains has its own query parameter for count=1

Reply:
FindOneWhereContains wrapped does not have its own count parameter, however. Might be worth checking if there are others as well.